captura/views.py

```diff
@@ -224,7 +227,153 @@ def create_estudio(request):
     if request.method == 'POST':
         familia = Familia.objects.create()
         Estudio.objects.create(capturista=request.user.capturista, familia=familia)
-        return redirect(reverse('home'))
+        return redirect(reverse('captura:familia', kwargs={'id_familia': familia.id}))
+
+
+@login_required
+@user_passes_test(is_capturista)
+def familia(request, id_familia):
```

Comment:
I believe this request is out of scope for this particular PR, due to the fact that a component like this should contain all of the parts of filling a study. I don't know about removing the edit button, as a capturista should be able to edit a study, including all of the sections from the left-side of the MER.

I believe the most sensible approach is the one I had discussed with Raul, of using a sidenav that allows the capturista to move between sections, of the study.